omnigibson/robots/fetch.py

```diff
@@ -387,6 +406,40 @@ def disabled_collision_pairs(self):
         return [
             ["torso_lift_link", "shoulder_lift_link"],
             ["torso_lift_link", "torso_fixed_link"],
+            ["torso_lift_link", "estop_link"],
+            ["base_link", "laser_link"],
+            ["base_link", "torso_fixed_link"],
+            ["base_link", "l_wheel_link"],
+            ["base_link", "r_wheel_link"],
+            ["base_link", "estop_link"],
+            ["torso_lift_link", "shoulder_pan_link"],
+            ["torso_lift_link", "head_pan_link"],
+            ["head_pan_link", "head_tilt_link"],
+            ["shoulder_pan_link", "shoulder_lift_link"],
+            ["shoulder_lift_link", "upperarm_roll_link"],
+            ["upperarm_roll_link", "elbow_flex_link"],
+            ["elbow_flex_link", "forearm_roll_link"],
+            ["forearm_roll_link", "wrist_flex_link"],
+            ["wrist_flex_link", "wrist_roll_link"],
+            ["wrist_roll_link", "gripper_link"],
+        ]
+
+    @property
+    def manipulation_link_names(self):
+        return [
+            "torso_lift_link", 
+            "head_pan_link", 
+            "head_tilt_link",  
+            "shoulder_pan_link", 
+            "shoulder_lift_link", 
+            "upperarm_roll_link", 
+            "elbow_flex_link", 
+            "forearm_roll_link", 
+            "wrist_flex_link", 
+            "wrist_roll_link", 
+            "gripper_link", 
+            "l_gripper_finger_link", 
+            "r_gripper_finger_link",
```

Comment:
Still not sure if it's the best idea to duplicate a bunch of repetitive information here. But I will ask @cremebrule be the final judge. If he is happy with this, I am fine as well.

Reply:
We're not happy about this, we have identified the fix for it but it's going to take a couple weeks for the team to implement it. You can track it on issue [#304](https://github.com/StanfordVL/OmniGibson/issues/304). This should still be merged for now.